include/loramesh.h

```diff
@@ -14,6 +16,8 @@ class LoraMesh {
   DataQueue<message_t>* txQueue;
   DataQueue<message_t>* rxQueue;
   Router* router;
+  Layer1Class* Layer1;
+  LL2Class* LL2;
```

Comment:
Class attributes must be camelCase, with first char lowercase.

Reply:
We rename layer1 to layer1 and LL2 to ll2